site/blog/2022/deploy-ghost-ec2-advanced.mdx

```diff
@@ -0,0 +1,192 @@
+---
+slug: deploy-ghost-ec2-advanced
+title: Advanced configuration for deploying Ghost on EC2
+date: 2022-12-28
+authors: [yrobla]
+---
+
+# Advanced configuration for deploying <a href="https://ghost.org/">Ghost</a> on EC2
+
+In our previous <a href="https://iasql.com/blog/deploy-ghost-ec2/">blogpost</a> we showed how to install <a href="https://ghost.org/">Ghost</a> in an EC2 instance, using a simple configuration. While it is ok
+for testing and development purposes, it was not ready for production. In order to be ready, it needs some essential features, that we will be illustrating on this new post.
```

Comment:
Nit: "essential features" feels too vague to me, can we just list out the things we need: domain name, SSL cert, HTTPS-to-HTTP load balancer, and RDS? (Because you could also consider an auto-scaling group for the instance behind the load balancer as something else needed for production, and we aren't covering that here.)